src/github/utils/config.ts

```diff
@@ -1,5 +1,4 @@
 import { Value } from "@sinclair/typebox/value";
-import { generateConfiguration } from "@ubiquibot/configuration";
```

Comment:
Is this referenced somewhere else? If not I believe this can also be removed from the `package.json`.

Reply:
I checked and it's not referenced anywhere so I'll remove it